scripts/rollup/forks.js

```diff
@@ -75,10 +75,13 @@ const forks = Object.freeze({
         return 'shared/forks/ReactFeatureFlags.persistent.js';
       case 'react-test-renderer':
         switch (bundleType) {
+          case RN_FB_DEV:
+          case RN_FB_PROD:
+          case RN_FB_PROFILING:
           case FB_WWW_DEV:
           case FB_WWW_PROD:
           case FB_WWW_PROFILING:
-            return 'shared/forks/ReactFeatureFlags.test-renderer.www.js';
```

Comment:
This rename might not be a great idea. Our current naming convention, although confusing, kind fo seems to be that "fb" stands for Facebook's "fbsource" and "www" stands for www.

We _could_ do a cleanup pass where we made them more cohesive (i.e. either "facebook", "facebook-www", or "facebook-fbsource") but unless we do a larger cleanup like that, I think the www feature flag file makes better sense with its original name.